app/controllers/assignments_controller.rb

```diff
@@ -52,4 +54,12 @@ def load_employees
       @employees = Employee.all
     end
 
+    def get_office
+      if params[:office_id].present?
+        @office = Office.find(params[:office_id])
+      else
+        @office = Office.first
+    end
+    end
```

Comment:
The office is very similar to the user since it will be relatively sticky from request to request. Let's rename `@office` to `@current_office`, create an `attr_reader` for it and a `helper` too. Finally, let's move all that stuff and the `get_office` method to the `ApplicationController`.